AirCasting/MainTabBarView.swift

```diff
@@ -104,7 +104,7 @@ private extension MainTabBarView {
                         if !searchAndFollow.isHidden && featureFlagsViewModel.enabledFeatures.contains(.searchAndFollow) && selectedSection.section == .following {
                             searchAndFollowButton
                         }
-                        if !reorderButton.isHidden && sessions.count > 1 && selectedSection.section == .following {
+                        if !reorderButton.isHidden && (reorderButton.reorderIsOn || sessions.count > 1) && selectedSection.section == .following {
```

Comment:
Please tell me, why do we need to have it here. I can't get it 😅

Reply:
I agree, maybe we should pull this part `(reorderButton.reorderIsOn || sessions.count > 1)` to a helper variable and give it a descriptive name? Or comment this line (but always prefer helper variables over comments since those are being checked by the compiler and will produce a warning if unused)

Reply:
This is added because otherwise, the "Finish" button disappeared if I cleared all of the sessions (or all except one). Now, the button stays even if there are less than 2 sessions as long as reordering is on. I don't know how could I call such variable

Reply:
But I can change the logic here and hopefully it will make it more clear :)